src/print-results.ts

```diff
@@ -251,3 +269,37 @@ function sumCounts(results: ESLint.LintResult[]): TodoFormatterCounts {
 function pluralize(word: string, count: number): string {
   return count === 1 ? word : `${word}s`;
 }
+
+function filterTodos(
+  results: ESLint.LintResult[],
+  options: TodoPrintOptions
+): ESLint.LintResult[] {
+  return results.reduce((acc, result) => {
+    const messages = result.messages as TodoResultMessage[];
+    if (messages.length === 0) {
+      return acc;
+    }
+
+    if (options.includeTodo) {
+      return [...acc, result];
+    }
```

Comment:
I'm not sure we should do this, since it's really unexpected for other formatters to be able to handle todos, so I'd likely just never include them.

```suggestion
```

Reply:
I left it in as, from an api perspective, if someone explicitly says `INCLUDE_TODOS=1` then I think they should get them.

But you're right, no other systems understand the concept of our todos here, so removing this is fine.